components/InputCard.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react'
+import { Button } from './theme/Button'
+import { MdInput } from './MdInput'
+
+const capitalizeFirst = (str: string) => {
+  return str.replace(/./, char => char.toUpperCase())
+}
+
+type InputCardProps = {
+  values: any
+  buttons: any
+  capitalizeTitle?: boolean
+  bgColor?: 'white' | 'none'
+  title?: string
+}
+
+type ButtonsProps = {
+  buttons: any
+  options: any
+}
+
+const Buttons: React.FC<ButtonsProps> = ({ buttons, options }) => {
+  const btnsList = Object.keys(buttons)
+
+  const btns = btnsList.map((title: string, i: number) => {
```

Comment:
It would be very difficult for any future engineer to understnd what is going on here. Need to create a story.